mscore/inspector/inspectorBase.h

```diff
@@ -40,6 +40,7 @@ struct InspectorItem {
       QWidget* w;
       // QToolButton* r;   // reset to default button (if any)
       QWidget* r;   // reset to default button (if any)
+      bool allowSetAsStyle = true;   // show S button in inspector
```

Comment:
Or intializing a struct's members inside a header files is not supported

Reply:
Then is there any way to solve this without adding this field to all `InspectorItem` defined in the solution?